server/service/integration_mdm_test.go

```diff
@@ -13266,6 +13266,144 @@ func (s *integrationMDMTestSuite) TestAppleDDMReconciliation() {
 	checkDDMSync(deviceThree)
 }
 
+func (s *integrationMDMTestSuite) TestAppleDDMStatusReport() {
```

Comment:
lol, you're right, that's a great idea. I know Sarah and Dante have PRs that modify this file, I will do this as soon as those land to avoid conflicts.